sig/ffi.rbs

```diff
@@ -14,9 +14,12 @@ module FFI
   CURRENT_PROCESS: current_process
   USE_THIS_PROCESS_AS_LIBRARY: current_process
 
+  class NotFoundError < LoadError
+  end
+
   private def self.custom_typedefs: () -> type_map
   def self.errno: () -> Integer
-  def self.errno=: (Integer) -> nil
+  def self.errno=: (Integer) -> void
```

Comment:
`xyz=` methods always return `self`, so IMHO the return type should be `Integer`. The same below.

Reply:
The `set_last_error` function in C actually returns `nil`, I'd rather not change that behaviour. In general, I think that `void` is a better approach for setter-like functions, as there's no agreed upon standard across ruby for what setters return, and void means "it returns something, but you should not depend on it".

If you disagree, I can at least make them all return `nil`.

Reply:
then the current implementation is in violation of that expectation, is that it? [LastError.errno is returning nil](https://github.com/ffi/ffi/blob/83f9135a13bf1c3f07bed832f0495e027b944cd9/ext/ffi_c/LastError.c#L182), and [FFI.errno calls it](https://github.com/ffi/ffi/blob/83f9135a13bf1c3f07bed832f0495e027b944cd9/lib/ffi/errno.rb#L35), so [this was never correct in the first place](https://github.com/ffi/ffi/blob/83f9135a13bf1c3f07bed832f0495e027b944cd9/sig/ffi.rbs#L18) (which may hint at sigs not being validated in CI).

I can try to do all those changes, but changing `FFI.errno` may cause compatibility issues, as current users of `ffi` may be relying on it (hence why I'm only changing the sigs).